src/pages/nft/token/_symbol.vue

```diff
@@ -151,7 +151,7 @@ export default Vue.extend({
   },
   computed: {
     returnLink(): string | Route {
-      return this.fromRoute && this.fromRoute.fullPath !== this.$route.fullPath && this.fromRoute.path !== "/withdraw" ? this.fromRoute : "/account/nft";
+      return this.fromRoute && this.fromRoute.fullPath !== this.$route.fullPath && !this.fromRoute.path.includes("/transaction/") ? this.fromRoute : "/account/nft";
```

Comment:
this.fromRoute && this.fromRoute.fullPath can be simplified w/t the ts ? or !.
!this.fromRoute.path.includes("/transaction/") IMO should be replaced w/t .startsWith